src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -37,6 +38,17 @@
   private final SparkMaxPIDController leftController = leftMotor.getPIDController();
   private final SparkMaxPIDController rightController = rightMotor.getPIDController();
 
+  private final ElevatorFeedforward upFF = new ElevatorFeedforward(kS, MG, kV, kA);
+
+  private final ElevatorFeedforward diagonalFF =
+      new ElevatorFeedforward(kS, MG * Math.cos(ANGLE), kV, kA);
+
+  private final TrapezoidProfile elevatorProfile =
+      new TrapezoidProfile(
+          new TrapezoidProfile.Constraints(kMaxV, kMaxA),
+          new TrapezoidProfile.State(HEIGHT, 0),
```

Comment:
can you add a comment signifying that this is the goal, and the second parameter is the initial?